Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -313,6 +314,18 @@ extension TopTabsViewController: Themeable {
         privateModeButton.imageView?.tintColor = privateModeButton.tintColor
         newTab.tintColor = UIColor.Photon.Grey40
         collectionView.backgroundColor = view.backgroundColor
+        */
```

Comment:
why did you comment all the old code and write it again with extra lines, you can just add new lines of code above